etc/keystone/mapping.json

```diff
@@ -26,6 +26,9 @@
                             },
                             {
                                 "name": "heat_stack_user"
+                            },
+                            {
+                                "name": "creator"
```

Comment:
I think we'll need a doc step to create this role?

Reply:
Here are a set of common roles that are present in a standard deployment.

``` shell
openstack --os-cloud mine role list -c Name
+-------------------------------+
| Name                          |
+-------------------------------+
| manager                       |
| admin                         |
| load-balancer_quota_admin     |
| load-balancer_admin           |
| load-balancer_member          |
| reader                        |
| heat_stack_user               |
| service                       |
| load-balancer_global_observer |
| load-balancer_observer        |
| member                        |
+-------------------------------+
```

I believe that we'll need a doc step to generate the `creator` role so that the mapping file is functional.

Reply:
For the load-balancer_member that's part of the current mapping file the actual role isn't created until octavia helm deploy creates it in the bootstrap, re; https://github.com/rackerlabs/genestack/blob/main/base-helm-configs/octavia/octavia-helm-overrides.yaml#L55

I have set up similar with barbican as that's the services the requires it, re; https://github.com/rackerlabs/genestack/blob/main/base-helm-configs/barbican/barbican-helm-overrides.yaml#L230

I can add additional docs if required but I believe the mapping file would be functional until the role is created just as it was with octavia/loadbalancer

Reply:
Thanks for the info, I appreciate it. Seems like this should be safe to go in as is.